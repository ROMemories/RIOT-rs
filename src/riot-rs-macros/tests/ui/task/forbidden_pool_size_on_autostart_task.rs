#![no_main]
#![feature(impl_trait_in_assoc_type)]
#![feature(type_alias_impl_trait)]
#![feature(used_with_arg)]

// FAIL: the `pool_size` parameter cannot be used on autostart task
#[riot_rs::task(autostart, pool_size = 4)]
async fn main() {}
