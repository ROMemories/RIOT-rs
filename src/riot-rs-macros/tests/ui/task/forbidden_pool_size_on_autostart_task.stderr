error: custom attribute panicked
 --> tests/ui/task/forbidden_pool_size_on_autostart_task.rs:7:1
  |
7 | #[riot_rs::task(autostart, pool_size = 4)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: pool size cannot be set on an `autostart` task
